From 0a54c1a45bc0c807c7622f5aae01595785993ac8 Mon Sep 17 00:00:00 2001
From: Liu Ying <victor.liu@nxp.com>
Date: Mon, 24 Jul 2017 16:32:48 +0800
Subject: [PATCH 2208/5242] MLK-16075-13 video: dpu: Add more sources for
 fetchdecode

commit  8fb5012229c4b2baccd51a254837ea03b5bbab04 from
https://source.codeaurora.org/external/imx/linux-imx.git

This patch adds more sources for fetchdecode.
The new sources are fetchdecode0, fetchdecode1 and fetchwarp2,
which are valid only on DPU v2.

Signed-off-by: Liu Ying <victor.liu@nxp.com>
Signed-off-by: Meng Li <Meng.Li@windriver.com>
---
 include/video/dpu.h |    3 +++
 1 file changed, 3 insertions(+)

diff --git a/include/video/dpu.h b/include/video/dpu.h
index a576bd1..2f36e79 100644
--- a/include/video/dpu.h
+++ b/include/video/dpu.h
@@ -248,8 +248,11 @@ enum dpu_irq {
 	FD_SRC_FETCHECO0	= ID_FETCHECO0,
 	FD_SRC_FETCHECO1	= ID_FETCHECO1,
 	FD_SRC_FETCHECO2	= ID_FETCHECO2,
+	FD_SRC_FETCHDECODE0	= ID_FETCHDECODE0,
+	FD_SRC_FETCHDECODE1	= ID_FETCHDECODE1,
 	FD_SRC_FETCHDECODE2	= ID_FETCHDECODE2,
 	FD_SRC_FETCHDECODE3	= ID_FETCHDECODE3,
+	FD_SRC_FETCHWARP2	= ID_FETCHWARP2,
 } fd_dynamic_src_sel_t;
 
 typedef enum {
-- 
1.7.9.5

